frontend/src/lib/components/LemonActionableTooltip/LemonActionableTooltip.scss

```diff
@@ -18,17 +22,28 @@
         }
     }
     .LemonActionableTooltip__body {
-        padding: 0.5rem;
+        padding: 0.25rem 0.5rem;
+        > * + * {
+            margin-top: 0.5rem;
+        }
     }
     .LemonActionableTooltip__footer {
         display: flex;
         justify-content: space-between;
+        padding: 0.5rem;
     }
-    .LemonActionableTooltip__buttons {
+    .LemonActionableTooltip__url_buttons {
```

Comment:
What do you think about making these `width: 100%` always? Would make the right padding of a button more predictable for users